### ISSUE TITLE
No changelog entry for 1.5.0

### ISSUE THREAD

**Issue by wjessop** [rkrage/pg_party#65]
There's no changelog entry for version 1.5.0 (which we're currently on making it rather moot) but at some point we will want to upgrade to any new version (I'm upgrading gems for the Ruby 3.0 keyword change and found `pg_party-1.5.0/lib/pg_party/adapter/postgresql_methods.rb:62: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call`). A changelog (and preferably a ommitment to semantic versioning) would help with the upgrade process by making it clearer when there are safe/dangerous changes we need to handle.

**Comment by wjessop** [comment]
Just hit this again looking to upgrade to 1.6. A changelog would make it a whole lot easier to upgrade!

**Comment by rkrage** [comment]
Yeah my bad. Honestly, this project is in maintenance mode at this point. I will continue to add support for new versions of Rails as time goes on, but I don't plan on adding any new features. FWIW, I have (and will continue) to follow semantic versioning to the best of my ability
